### PR TITLE
adding the namespace for ArrayInput

### PR DESCRIPTION
**Pull request by nodefortytwo**
Makes the documentation a little cleaner, easier to copy

### CODE REVIEW

File: console/calling_commands.rst [hunk 1]
```diff
@@ -10,6 +10,9 @@ generating Doctrine2 proxies, dumping Assetic assets, ...).
 
 Calling a command from another one is straightforward::
 
+    use Symfony\Component\Console\Input\ArrayInput
```
Comment (chalasr) [inline console/calling_commands.rst]: missing semicolon

File: console/calling_commands.rst [hunk 2]
```diff
@@ -10,6 +10,9 @@ generating Doctrine2 proxies, dumping Assetic assets, ...).
...
+    // ...
+
     protected function execute(InputInterface $input, OutputInterface $output)
```
Comment (chalasr) [inline console/calling_commands.rst]: If we do it for `ArrayInput`, shouldn't we do it for these too?

**Review by chalasr — CHANGES_REQUESTED** [review]

**Comment by xabbuh** [comment]
👍 I think it makes sense to document it this way. All other classes should indeed be known at this point.

Status: Reviewed

**Comment by xabbuh** [comment]
Thank you @nodefortytwo.

**Pull request closed without merging**